fmpz/get_nmod.c

```diff
@@ -0,0 +1,58 @@
+/*
+    Copyright (C) 2021 Daniel Schultz
+
+    This file is part of FLINT.
+
+    FLINT is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.  See <https://www.gnu.org/licenses/>.
+*/
+
+#include "fmpz.h"
+
+mp_limb_t fmpz_get_nmod(const fmpz_t aa, nmod_t mod)
```

Comment:
How is this different to fmpz_fdiv_ui?

Reply:
Doesn't call %

Reply:
So it's faster? If so why not speed up the other function so everything benefits, rather than create a new one?

Reply:
What other functions? fmpz_fdiv_ui doesn't take a precomputed inverse...

Reply:
Oh I see